rasa/importers/rasa.py

```diff
@@ -25,17 +26,37 @@ def __init__(
         domain_path: Optional[Text] = None,
         training_data_paths: Optional[Union[List[Text], Text]] = None,
     ):
-        if config_file and os.path.exists(config_file):
-            self.config = io_utils.read_config_file(config_file)
-        else:
-            self.config = {}
 
         self._domain_path = domain_path
 
         self._story_files, self._nlu_files = data.get_core_nlu_files(
             training_data_paths
         )
 
+        if config_file and os.path.exists(config_file):
+            config = io_utils.read_config_file(config_file)
```

Comment:
I'd move the whole `if` leg to a function within `autoconfig` to avoid that logic / responsibilities of the `autoconfig` module leak into the importer.